cmd/ifparse.c

```diff
@@ -47,24 +47,18 @@ set_allow_undefined(const char *arg)
 	allow_undefined = true;
 }
 
-#ifdef CONFIG_YAML
-static bool use_yaml = false;
+static const char *output_fmt = "ifupdown";
 
 static void
-set_yaml(const char *arg)
+set_output_fmt(const char *arg)
 {
-	(void) arg;
-
-	use_yaml = true;
+	output_fmt = arg;
```

Comment:
Shouldn't we `strcmp()` for `ifupdown` or `yaml` here?

Reply:
nah it’s handled later by the lookup